### PR TITLE
Blackjaxified `hierarchical_binom_rats.ipynb`

### PR DESCRIPTION
**Pull request by karm-patel**
## Description

Replaced pymc3 implementation with blackjax.
 

### Figure Number
Book2 | Fig 3.13

#### Before PR
image

#### After PR
image

### Issue 
#879

## Checklist

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

### CODE REVIEW

File: notebooks/book2/03/hierarchical_binom_rats.ipynb [hunk 1]
```diff
@@ -0,0 +1,1831 @@
+{
```
Comment (murphyk) [inline notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: <p>English needs rewriting. Just say this</p><p>`</p><pre class="ql-syntax" spellcheck="false">We fit a hierarchical beta-binomial model to some count data derived from rat survival.  Based on&nbsp;https://docs.pymc.io/notebooks/GLM-hierarchical-binominal-model.html
</pre><p>`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (murphyk) [inline notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    import jax
</pre><p>Add link to pymc version as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (murphyk) [inline notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #40.            7,
</pre><p>Can you reformat this data vector so it takes less space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (murphyk) [inline notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.        [
</pre><p>Can you reformat this data vector so it takes less space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (murphyk) [reply notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: Correct URL: https://www.pymc.io/projects/examples/en/latest/generalized_linear_models/GLM-hierarchical-binomial-model.html

Comment (karm-patel) [reply notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: Unfortunately, there was no way given that how black format list. So I have moved data to `probml_data`.

**Comment by karm-patel** [comment]
Dr. @murphyk
I've added the tutorial `change_of_variable_hmc.ipynb` in this PR. 
This notebook is merged in blackjax repo.  See this PR: https://github.com/blackjax-devs/blackjax/pull/254

File: notebooks/book2/03/hierarchical_binom_rats.ipynb (hunk 1 above)
Comment (murphyk) [reply notebooks/book2/03/hierarchical_binom_rats.ipynb:1]: Yes I suspect this was a black problem. Is there no way to tell black to ignore formatting rules for a given line?
This comes up in several notebooks where we hard-code a small dataset. It seems a bit of a pain to have to store the data in probml_data.

**Pull request merged**